src/components/RowGridLayoutList.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="row-grid-layout-list" :class="classes.root">
+    <template v-for="item in props.items" :key="item.id">
+      <slot :item="item" />
+    </template>
+
+    <template v-if="empty">
+      <slot name="empty" />
+    </template>
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { computed } from 'vue'
+  import { activeViewMode } from '@/utilities/activeViewMode'
+
+  const props = defineProps<{
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    items: any[],
```

Comment:
`unknown` doesn't work here?

Reply:
I can never keep straight when it won't work...

Reply:
It doesn't, it's bugging me cause any isn't correct. TS throws an error when trying to cast/infer the type in the slot caller if this is `unknown`